client/src/pages/restaurant-list.js

```diff
@@ -1,22 +1,101 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-import PropTypes from 'prop-types';
+import { Container } from 'react-bootstrap';
+import Pagination from 'react-bootstrap-pagination-logic';
+import ButtonToolbar from 'react-bootstrap/ButtonToolbar';
+import Spinner from 'react-bootstrap/Spinner';
+import Api from '../services/api';
+import RestaurantCardResults from '../components/restaurant/restaurant-card-results';
 
 class ListRestaurant extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      totalPages: 0,
+      number: 0,
+      restaurants: [],
+      isFetching: false,
+    };
+    this.handlePageChange = this.handlePageChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.getAll(this.state.number);
+  }
+
+  getAll(props) {
+    Api.getAll(`restaurants?page=${props}&size=21`)
```

Comment:
Try to avoid to use "magic" constants. You can create class property e.g. `pageSize = 21` and use this named constant.